content/docs/instrumenting/clientlibs.md

```diff
@@ -21,6 +21,7 @@ HTTP endpoint on your application’s instance:
 Unofficial third-party client libraries:
 
 * [Bash](https://github.com/aecolley/client_bash)
+* [Clojure](https://github.com/soundcloud/prometheus-clj) (wraps the Java client library)
```

Comment:
> The "directly instrumented software" section is perhaps a bit misnamed, as gokit and go-metrics don't really fit. 

It's a little odd, but I'd say go-kit is directly instrumented. go-metrics is more a collector.